DOD-ISD-Verbs/dodisdverbs.jsonld

```diff
@@ -0,0 +1,4729 @@
+{
+	"@context": "https://w3id.org/xapi/profiles/context",
+	"id": "https://w3id.org/xapi/profiles/dod-isd",
+	"type": "Profile",
+	"conformsTo": "https://w3id.org/xapi/profiles#1.0",
+	"prefLabel": {
+		"en": "DOD ISD"
+	},
+	"definition": {
+		"en": "Verbs and extensions for use in creating DOD IMI, per MIL-HDBK-29612-1A."
+	},
+	"seeAlso": "MIL-HDBK-29612-1A",
+	"versions": [
+		{
+			"id": "1.0",
+			"generatedAtTime": "2017-10-24"
+		}
+	],
+	"author": {
+		"type": "Organization",
+		"name": "CAE USA"
+	},
+	"concepts": [
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/advised",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
```

Comment:
This must use `type` instead of `@type` (in the profile context, `type` is aliased to `@type`, and the specification requires the use of `type`)